pkg/daemon/kernelargs.go

```diff
@@ -45,7 +39,7 @@ func isArgTunable(arg string) (bool, error) {
 	if os.IsRHCOS() {
 		return tuneableRHCOSArgsAllowlist[arg], nil
 	} else if os.IsFCOS() {
-		return tuneableFCOSArgsAllowlist[arg], nil
+		return true, nil
```

Comment:
For future reference, can you add a one line comment here providing reasoning of why we are allowing everything on FCOS

Reply:
maybe add a FCOS specific note here: https://github.com/openshift/machine-config-operator/blob/master/docs/MachineConfiguration.md#kernelarguments

Reply:
I've looked at the code path here a bit more closely, and this restriction only seems to apply to the `/etc/pivot/kernel-args` file interface for setting kargs, not however to the in-MC way of doing it which we are now using in OKD. This makes this change unnecessary. Closing as obsolete.

Reply:
that's right. This code exist to tweak default kargs shipped with OS and is useful during node first boot.